lib/cylc/cfgspec/gcylc.py

```diff
@@ -173,8 +183,24 @@ def transform(self):
         cfg['themes'] = cfg_themes
 
     def check(self):
-        # check initial view config
         cfg = self.get(sparse=True)
+
+        # check window size config
+        if 'window size' in cfg:
+            fail = False
+            if len(cfg['window size']) != 2:
+                print >> sys.stderr, ("WARNING: window size requires two "
+                                      "values (x, y). Using default.")
+                fail = True
+            elif cfg['window size'][0] < 0 or cfg['window size'][1] < 0:
+                print >> sys.stderr, ("WARNING: window size values must be "
+                                      "positive. Using default.")
+                fail = True
+            # TODO: check for daft window sizes? (10, 5), (80000, 5000) ?
```

Comment:
On my desktop it would appear so, though I can't find anything in the gtk documentation to confirm this.